core/src/main/java/org/apache/iceberg/rest/RESTSessionCatalog.java

```diff
@@ -168,15 +168,22 @@ public void initialize(String name, Map<String, String> unresolved) {
     Map<String, String> baseHeaders = configHeaders(mergedProps);
 
     this.sessions = newSessionCache(mergedProps);
-    this.refreshAuthByDefault =
+    boolean keepTokenRefreshed =
         PropertyUtil.propertyAsBoolean(
             mergedProps,
-            CatalogProperties.AUTH_DEFAULT_REFRESH_ENABLED,
-            CatalogProperties.AUTH_DEFAULT_REFRESH_ENABLED_DEFAULT);
+            OAuth2Properties.TOKEN_REFRESH_ENABLED,
+            OAuth2Properties.TOKEN_REFRESH_ENABLED_DEFAULT);
```

Comment:
I'm fine not keeping compatibility with the property since I really doubt anyone is using it.

Reply:
that was my thought as well for these